langDir/all_lang.json

```diff
@@ -14,6 +14,14 @@
         "href": "https://kusama.subscan.io/extrinsic/6095478-1"
       }
     },
+    "kusama_github": {
+      "open_issues": {
+        "href": "https://github.com/kodadot/nft-gallery/issues"
+      },
+      "contributing_guide": {
+        "href": "https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md"
+      }
```

Comment:
Hey! you should move this to `kodadot_social: {}`

Reply:
done!